test/e2e/tests/settings/auto-lock.spec.js

```diff
@@ -41,12 +40,11 @@ describe('Auto-Lock Timer', function () {
           '[data-testid="advanced-setting-auto-lock"] button',
         );
         // Verify the wallet is locked
-        const pageTitle = await driver.findElement(
-          '[data-testid="unlock-page-title"]',
-        );
-        const unlockButton = await driver.findElement('.unlock-page button');
-        assert.equal(await pageTitle.getText(), 'Welcome back!');
-        assert.equal(await unlockButton.isDisplayed(), true);
+        driver.waitForSelector({
+          css: '[data-testid="unlock-page-title"]',
+          text: 'Welcome back!',
+        });
```

Comment:
missing `await`

Reply:
whoaa nice catch :heart_eyes: !! Thank you!! fixed :pray: